gguf-py/gguf/gguf_writer.py

```diff
@@ -704,6 +704,9 @@ def add_middle_token_id(self, id: int) -> None:
     def add_eot_token_id(self, id: int) -> None:
         self.add_uint32(Keys.Tokenizer.EOT_ID, id)
 
+    def add_swin_norm(self, value: bool) -> None:
+        self.add_bool(Keys.LLM.SWIN_NORM.format(arch=self.arch), value)
+
```

Comment:
This could be moved between `add_expert_weights_scale` and `add_layer_norm_eps`.

Otherwise it feels a bit out of place near the tokenizer options.